ansible/roles/operator-pipeline/templates/openshift/tasks/read-config.yaml

```diff
@@ -41,7 +41,13 @@ spec:
 
         CONFIG_PATH="$PKG_PATH/ci.yaml"
 
-        if [[ ! -f "$CONFIG_PATH" ]]; then
+        if [[ ! -d "$PKG_PATH" ]]; then
+            echo "Operator directory is removed."
```

Comment:
Non-blocking: Does it make a sense to indicate here which operator has been removed?